src/lib.rs

```diff
@@ -853,4 +873,105 @@ mod tests {
 
         assert_eq!(start.as_millis() as usize, session.speech_start_ms.unwrap());
     }
+
+    fn get_audio_sample(audio_file: impl AsRef<Path>, config: &VadConfig) -> Result<Vec<f32>> {
```

Comment:
Comment: it's a test you can just unwrap instead of returning results and that's typically easier to debug with because you can't risk losing the location of the issue